### PR TITLE
Fixes #855 - Added QSY column to SMR lists

### LINKED ISSUE #855: Add QSY column to taxi out lists
## What is the current situation?
QSY not in taxi-out list

## What is the proposed change?
Add QSY column to taxi-out list (not shown by default)

### PR DESCRIPTION
**Pull request by kristiankunc**
Fixes #855 

# Summary of changes

Added QSY col to the combined dep list
image

### CODE REVIEW

File: UK/Data/Settings/Lists_SMR.txt [hunk 1]
```diff
@@ -95,6 +95,7 @@ m_Column:NFREQ:7:0:107:0:0:0:UK Controller Plugin:::1:0.0
 m_Column:RLS:3:1:124:9012:9015:1:UK Controller Plugin:UK Controller Plugin:UK Controller Plugin:1:0.0
 m_Column::6:0:125:9014:9014:1:UK Controller Plugin:UK Controller Plugin:UK Controller Plugin:1:0.0
 m_Column:PRE:3:1:127:9017:9018:0:UK Controller Plugin:UK Controller Plugin:UK Controller Plugin:1:0.0
+m_Column:QSY:7:0:107:0:0:0:UK Controller Plugin:::1:0.0
```
Comment (luke11brown) [inline UK/Data/Settings/Lists_SMR.txt:98]: Agree with this, is it hidden by default? Screenshot would help.

File: UK/Data/Settings/Lists_SMR.txt [hunk 2]
```diff
@@ -205,6 +206,7 @@ m_Column:ASSR:5:1:60:9000:9022:0::UK Controller Plugin:UK Controller Plugin:0:0.
 m_Column:RLS:3:1:124:9012:9015:1:UK Controller Plugin:UK Controller Plugin:UK Controller Plugin:1:0.0
 m_Column::6:0:125:9014:9014:1:UK Controller Plugin:UK Controller Plugin:UK Controller Plugin:1:0.0
 m_Column:PRE:3:1:127:9017:9018:1:UK Controller Plugin:UK Controller Plugin:UK Controller Plugin:1:0.0
+m_Column:QSY:7:0:107:0:0:0:UK Controller Plugin:::1:0.0
```
Comment (luke11brown) [inline UK/Data/Settings/Lists_SMR.txt:209]: Disagree with this change. The next frequency is only applicable to TWR controller as it's the next airborne frequency. Having it only in the take-off lists encourages controllers to use the GND STS field correctly.

**Review by luke11brown — CHANGES_REQUESTED** [review]

File: UK/Data/Settings/Lists_SMR.txt (hunk 2 above)
Comment (kristiankunc) [reply UK/Data/Settings/Lists_SMR.txt:209]: I see, my idea was that someone might still use the departure list so I wanted to add the functionality there as well.

File: UK/Data/Settings/Lists_SMR.txt (hunk 1 above)
Comment (kristiankunc) [reply UK/Data/Settings/Lists_SMR.txt:98]: Indeed hidden by default
image

**Review by luke11brown — APPROVED** [review]

**Pull request merged**